### PR TITLE
Allow pushing all or multiple queries with stmocli push

### LINKED ISSUE #6: Add support for pushing all known queries
A simple `stmocli push` should push all queries registered in the config.

Alternatively, support multiple filename arguments to `push`, allowing `stmocli push *`.

### PR DESCRIPTION
**Pull request by kewisch**
Sometimes you just want to push all the queries. With `stmocli push` you can push all tracked
queries, or you can pass specific files to be pushed. [fixes #6]

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -176,12 +187,67 @@ def test_push_tracked(runner):
...
+            ])
+
+        m = hashlib.md5(query_62375_after.encode("utf-8"))
+        expected_output_62375 = "Query ID 62375 updated with content from 62375.sql " + \
+                                "(md5 {})".format(m.hexdigest())
+        m = hashlib.md5(query_49741_after.encode("utf-8"))
+        expected_output_49741 = "Query ID 49741 updated with content from 49741.sql " + \
+                                "(md5 {})".format(m.hexdigest())
+
+        assert push_result.output.strip() == expected_output_49741 + "\n" + expected_output_62375
```
Comment (harterrt) [inline tests/test_cli.py]: Looks like this test is fragile. The CI for python 2.7 handles 62375 before handling 49741 so these strings don't match. Instead, we can either: 

1. Exclusively verify that each MD5 hash is included in the output
2. Make this more flexible by splitting the actual output on newlines and doing a `set` equals

File: tests/test_cli.py [hunk 2]
```diff
@@ -176,12 +187,67 @@ def test_push_tracked(runner):
...
+
+def test_push_all(runner):
```
Comment (harterrt) [inline tests/test_cli.py]: Need two blank lines before this function definition. From the CI:

```
tests/test_cli.py:218:1: E302 expected 2 blank lines, found 1
```

Comment (kewisch) [reply tests/test_cli.py]: Heh, last minute changes. Will fix this.

File: tests/test_cli.py (hunk 1 above)
Comment (kewisch) [reply tests/test_cli.py]: I'll go with the set() approach, thanks for the suggestion

**Comment by harterrt** [comment]
Awesome! Thanks!

**Pull request merged**